lib/matplotlib/backends/backend_pgf.py

```diff
@@ -976,8 +951,14 @@ def __init__(self, *args):
         FigureManagerBase.__init__(self, *args)
 
 
-FigureCanvas = FigureCanvasPgf
-FigureManager = FigureManagerPgf
+@_Backend.export
+class _BackendPgf(_Backend):
+    FigureCanvas = FigureCanvasPgf
+    FigureManager = FigureManagerPgf
+
+
+def draw_if_interactive():
```

Comment:
Either way is *technically* fine (this def will just override the one exported by the decorator) but I agree a staticmethod is better.